qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -469,6 +469,59 @@ def process_dfe(self, ids: List[str]) -> float:
         """
         return self._client.process_dfe(ids)
 
+    def submit_aces(
```

Comment:
it looks like this is the same in css.Service and qss.SuperstaqProvider - should we just move it to `gss.service.Service` instead?

also we should add it to `qss.SuperstaqBackend` (which would just be a wrapper for `self._provider(target=self.name(), ...)`)

Reply:
> it looks like this is the same in css.Service and qss.SuperstaqProvider - should we just move it to gss.service.Service instead?

sounds good

> also we should add it to qss.SuperstaqBackend (which would just be a wrapper for self._provider(target=self.name(), ...))

will do

Reply:
done!